texi2rst.py

```diff
@@ -1666,6 +1670,13 @@ def test_valid_option_ref(self):
         out = self.make_rst_string(doc)
         self.assertEqual(u':option:`--some-opt`', out)
 
+    def test_option_ref_with_var(self):
+        xml_src = '<para>This is similar to how <option>-Walloca-larger-than=</option><var>byte-size</var> works.</para>'
+        doc = from_xml_string(xml_src)
+        doc = fixup_option_refs(doc)
+        out = self.make_rst_string(doc)
+        self.assertEqual('This is similar to how :option:`-Walloca-larger-than` = byte-size works.\n\n', out)
```

Comment:
Should the .rst output do something to the {{{= byte-size}}} rather than having it as plain text?  It's enclosed in a <var> element; seems a shame to just drop it on the floor.

Reply:
Yep, it's work-in-progress..